Assets/Script/Physics/PlayerController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+
+namespace PlayerPhysics
+{
+    [RequireComponent(typeof(Animator))]
+    public class PlayerController : MonoBehaviour
+    {
+        Animator MainCharecter;
+
+        float player_Speed;
+
+        Vector3 player_Scale;
```

Comment:
brought them out of update to stop the script from creating the variables on every frame.